### ISSUE TITLE
Create sub-sections in md files

### ISSUE THREAD

**Issue by dvidsilva** [gloriajs/gloria#104]
Currently, all the content in the md file is parsed and added to a property called content in the data.

Create a system that allow us to add new sections, and those sections can be consumed by the layouts, includes, etc.

For example, let's say my md file has
```
==content
This is text
===/content

===image
https://imgur.com/images
===/image
```

Then on the layout i could have something like:

```
<img src={{page.image}}/>
<div class=container>
    {{page.content}}
</div>
```

Something like this can be accomplished with YAML currently, but is very limited. This feature would add more flexibility to the content types.

**Comment by AWulkan** [comment]
So basically you want to be able to simulate having different input fields, like a normal CMS might have?
Like a header, body and footer part of an article? Sounds cool.  

As for the image link I think it makes more sense to put it in the metadata at the top of the file, or am I wrong?

**Comment by dvidsilva** [comment]
Yes, exactly, the image example is too simple, but it can help with much more complex things, like a whole new piece of content, sub menus, and things like that.

Specially if we're able to implement hierarchies in layouts and includes.
